docs/administration.building-blocks.md

```diff
@@ -27,6 +27,20 @@ In the Admin Area, click on the "Building Block" item in the sidebar under "Plat
 
 > meshStack supports Terraform up to version 1.5.5 and supports OpenTofu versions starting with 1.6.0.
 
+### Building Block Types
```

Comment:
Please use lower case spelling for building blocks etx

Reply:
This is a title. It should be upper case